specification/communication/data-plane/PhoneNumbers/preview/2023-10-01-preview/phonenumbers.json

```diff
@@ -1467,6 +1864,270 @@
         }
       }
     },
+    "PhoneNumbersBrowseRequest": {
+      "required": [
+        "phoneNumberType"
+      ],
+      "type": "object",
+      "properties": {
+        "phoneNumberType": {
+          "description": "Represents the number type of the offering.",
+          "enum": [
+            "geographic",
+            "tollFree"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "PhoneNumberType",
+            "modelAsString": true
+          }
+        },
+        "capabilities": {
+          "$ref": "#/definitions/PhoneNumberBrowseCapabilitiesRequest"
+        },
+        "assignmentType": {
+          "description": "Represents the assignment type of the offering. Also known as the use case.",
+          "enum": [
+            "person",
+            "application"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "PhoneNumberAssignmentType",
+            "modelAsString": true
+          }
+        },
+        "phoneNumberPrefixes": {
+          "description": "The phone number prefix to match. If specified, the search will be limited to phone numbers that start with the any of the given prefixes.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "example": [
+            "1425",
+            "1626"
+          ]
+        }
+      }
+    },
+    "PhoneNumberBrowseCapabilitiesRequest": {
+      "description": "The minimum desired capabilities for the browse operation request.",
+      "type": "object",
+      "properties": {
+        "calling": {
+          "description": "Capability value for calling.",
+          "enum": [
+            "none",
```

Comment:
Rather than "none" values, you should treat the absence of these various enum types (like `sms` below) as "none".